Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gstools-core"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
Why a version bump? Since version `0.2.0` has no tag and no release on pypi yet, we should keep v0.2.0. Also GSTools docs are referring to v0.2 as a dependency.

Reply:
Ah thanks! I remember that I forgot to create a v0.2.0 tag ;-)